### PR TITLE
name change

### PR DESCRIPTION
**Pull request by steftheo**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -3,8 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-
-    <title>Emberland</title>
+    <title>Funland!</title>
```
Comment (rtablada) [inline app/index.html:6]: You undid my title change?

Comment (steftheo) [inline app/index.html]: hahahahahahahahahahahah GET GUD

Comment (steftheo) [inline app/index.html]: hi ryan

Comment (steftheo) [inline app/index.html]: hi ryan

Comment (steftheo) [inline app/index.html]: hi ryan

**Pull request merged**